sympy/tensor/indexed.py

```diff
@@ -159,6 +159,14 @@ def __new__(cls, base, *args, **kw_args):
 
         return Expr.__new__(cls, base, *args, **kw_args)
 
+    @property
+    def name(self):
+        indices_str = "["
+        for index in self.args[1:-1]:
+            indices_str += str(index) + ","
+        indices_str += str(self.args[-1]) + "]"
+        return self.base.name + indices_str
```

Comment:
Would it suffice to return `str(self)`?

Reply:
Yes, it does. Missed that earlier. Updated now.